akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -1981,6 +2025,8 @@ class BusLogging(val bus: LoggingBus, val logSource: String, val logClass: Class
     bus.publish(Error(cause, logSource, logClass, message, mdc))
   protected def notifyWarning(message: String): Unit =
     bus.publish(Warning(logSource, logClass, message, mdc))
+  protected def notifyWarning(cause: Throwable, message: String): Unit =
+    bus.publish(Warning(cause, logSource, logClass, message, mdc))
```

Comment:
Funny that we had prepared for it with that `Warning4` class, but not exposing it in the LoggingAdapter. Must have been an oversight.